assets/src/edit-story/components/keyboardShortcutsMenu/constants.js

```diff
@@ -19,6 +19,11 @@
  */
 import { __ } from '@wordpress/i18n';
 
+/**
+ * Internal dependencies
+ */
+import { Up, Down } from './styled';
```

Comment:
Are we shifting to start using keeping `styled` separately?

Reply:
It's been used a few times. I don't think we should do this automatically, but if need be, yes.